pkg/workload/tpcc/partition.go

```diff
@@ -377,9 +377,21 @@ func partitionHistory(db *gosql.DB, cfg zoneConfig, wPart *partitioner) error {
 
 // replicateItem creates a covering "replicated index" for the item table for
 // each of the zones provided. The item table is immutable, so this comes at a
-// negligible cost and allows all lookups into it to be local.
-func replicateItem(db *gosql.DB, cfg zoneConfig) error {
-	for i, zone := range cfg.zones {
+// negligible cost and allows all lookups into it to be local. If there are no
+// zone it assumes that each partition corresponds to a rack.
+func replicateItem(db *gosql.DB, cfg zoneConfig, partitions int) error {
+	var constraints []string
+	if len(cfg.zones) == 0 {
```

Comment:
Two nits:

1. switch this condition to `if len(cfg.zones) > 0 {` so that it mirrors the logic in `configureZone`.
2. replace `partitions` with `*partitioner` so that `replicateItem` remains consistent with all the other `partitionTable` functions.